### DISCUSSION TITLE
Cross Cluster Search and central "vulnerability detection" problem

### DISCUSSION THREAD

**Discussion by stepkinn** [wazuh/wazuh#26462] [Q&A]
Hello,

I followed the CrossClusterSearch documentation from wazuh (https://wazuh.com/blog/managing-multiple-wazuh-clusters-with-cross-cluster-search/) and I have a problem with the centralization of vulnerability detections.

All the vulnerabilities are not centralized in a single dashboard, I only have the vulnerabilities on the site selected in API (menu from top). When I select "EcoleA", I have vulns from EcoleA only. When I select "EcoleB", I have vulns from EcoleB only.

In my search, in Dashboard management> Dashboard management > Index patterns I have the index *:wazuh-states-vulnerabilities-* twice. Is it normal ? (I deleted them and recreate one *:wazuh-states-vulnerabilities-* and I have already two *:wazuh-states-vulnerabilities-*) 

![wazuh-cross-cluster-search-and-central-vulenrability-v0-veb8y3vl4ovd1](https://github.com/user-attachments/assets/c5cbb14e-84c4-4bd9-8e1b-23abd07a6fe2)

Wazuh : 4.8.2 (like in the documentation)
Agents : 2 on each manager
OS : Debian 12

Thanks.

**Comment by MiguelazoDS** [comment]
Hi @stepkinn, 

Did you add the pattern in the Dashboard -> App settings section? 

I worked on a similar case and we found a solution for this because the dashboard could not see the index of the indexer connected to the manager. 

This is what should do. Add this `*:wazuh-states-vulnerabilities-*` in Dashboard Management -> App Settings

![image](https://github.com/user-attachments/assets/dd80926c-78a0-43fe-a5b0-cd849281ea02)

For more information 

https://github.com/wazuh/wazuh-documentation/issues/7842#issuecomment-2394466974

**Reply by FrankyFire** [reply]
Hi!
Thank you for this guide. The section "Vulnerabilities" is not available anymore in App settings in current versions and leads to broken dashboards with CCS. Is there a new way to configure this?

**Reply by MiguelazoDS** [reply]
Hi @FrankyFire, mmm that's weird they removed that, please contact to the dashboard team through the channel `#wazuh-dashboard` in Slack. Regards.

https://wazuh.com/community/

**Reply by FrankyFire** [reply]
For anyone trying to fix the issue with the Vulnerability Detection Dashboard with a more recent version:
1. Go to Dashboard Management -> Index Patterns
2. Remove wazuh-states-vulnerabilities-* pattern
3. Create a new Index Pattern *:wazuh-states-vulnerabilities-*.
Important: In the second window, select NO Time field (this is wrong in the Blog post about CCS), Show advanced settings and set the ID to wazuh-states-vulnerabilities-*

In our case it fixed the issues. If there's still something wrong, it might be necessary to clear your browser cache.

**Comment by sebasfalcone** [comment]
Inactive thread, please re-open it if needed
